app/src/main/java/com/github/h3lp3rs/h3lp/MedicalCardActivity.kt

```diff
@@ -94,7 +90,6 @@ class MedicalCardActivity : AppCompatActivity() {
     /**
      * create the field for the birth year with input check
      */
-    @RequiresApi(Build.VERSION_CODES.N)
```

Comment:
This was asked by Calendar class. Are you sure it won't cause problem on old api version without the required instruction?

Reply:
No, I tried removing it and it worked :D
You can see the build succeeds!